clang-tools-extra/include-cleaner/lib/WalkAST.cpp

```diff
@@ -32,6 +33,11 @@
 
 namespace clang::include_cleaner {
 namespace {
+bool isImplicitOperatorNewDelete(OverloadedOperatorKind OpKind) {
```

Comment:
Nit: this function only checks whether `OpKind` is of the new/delete kind. Having `Implici`t in the function name seems a bit misleading to me -- I think we can simply remove it.